### PR TITLE
[INFRA-1810] Deploy jenkinsadmin ircbot on kubernetes cluster

### PR DESCRIPTION
**Pull request by olblak**
This PR deploy the jenkins admin ircbot on the kubernetes cluster.
It also stop:
* jenkinsadmin docker container from OKRA
* Stop bind service

### CODE REVIEW

File: dist/profile/manifests/jenkinsadmin.pp [hunk 1]
```diff
@@ -1,6 +1,14 @@
 #
 # IRC bot that runs most project related tasks
 # containerized in https://github.com/jenkins-infra/ircbot
+
+#############################################################
+##                                                         ##
+##  THIS CLASS IS DEPRECATED IN FAVOR OF                   ##
+##  profile::kubernetes::resources::chatbot_jenkinsadminl  ##
```
Comment (rtyler) [inline dist/profile/manifests/jenkinsadmin.pp]: not important, but a type in the resourec name "jenkinsadminl"

File: dist/profile/manifests/kubernetes/resources/chatbot_jenkinsadmin.pp [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        'jira'   => base64('encode', $jira_credentials , 'strict'),
+      },
+      resource   => 'chatbot_jenkinsadmin/secret.yaml'
+    }
+
+    profile::kubernetes::apply { "chatbot_jenkinsadmin/deployment.yaml on ${context}":
+      context    => $context,
+      parameters => {
+        'image_tag'     =>  $image_tag,
+        'nick_password' => $nick_password
```
Comment (rtyler) [inline dist/profile/manifests/kubernetes/resources/chatbot_jenkinsadmin.pp]: Is there a reason this isn't being set as a secret too?

File: dist/profile/templates/kubernetes/resources/chatbot_jenkinsadmin/secret.yaml.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: jenkinsadmin
+  namespace: chatbot
+data:
+  .github: |
```
Comment (rtyler) [inline dist/profile/templates/kubernetes/resources/chatbot_jenkinsadmin/secret.yaml.erb]: There must be some YAML feature being used here that I'm not aware of, why does the key have a leading `.`?

File: dist/profile/manifests/jenkinsadmin.pp (hunk 1 above)
Comment (olblak) [reply dist/profile/manifests/jenkinsadmin.pp]: :trumpet:

File: dist/profile/templates/kubernetes/resources/chatbot_jenkinsadmin/secret.yaml.erb (hunk 3 above)
Comment (olblak) [reply dist/profile/templates/kubernetes/resources/chatbot_jenkinsadmin/secret.yaml.erb]: It's not a YAML feature but a kubernetes specificity, a secret can either be mounted as a environment variable or mounted as a volume into a pod. If it's mounted as volume, by default it mounts all secret information into a mounted directory where each secret key represent a filename and the file contents is defined by the value. In this case .github is mounted in /home/chatbot/.github. But we can still decided to only mount several keys or change the destination filename, which is what I will do as I am moving the nick_password to that secret and I don't need to mount it as file

File: dist/profile/manifests/kubernetes/resources/chatbot_jenkinsadmin.pp (hunk 2 above)
Comment (olblak) [reply dist/profile/manifests/kubernetes/resources/chatbot_jenkinsadmin.pp]: I'll update it

**Review by rtyler — APPROVED** [review]

**Pull request merged**